src/benchie/benchmark.ts

```diff
@@ -12,6 +12,8 @@ export interface IMethodBuilder {
 export interface IScenarioBuilder extends IMethodBuilder {
   beforeEach(task: Task): void;
   beforeAll(task: Task): void;
+  afterAll(task: Task): void;
+  afterEach(task: Task): void;
```

Comment:
```
  constructor(params: IScenarioParams) {
    this.beforeAll = params.beforeAll;
    this.beforeEach = params.beforeEach;
    this.method = params.method;
    this.body = params.body;
    this.title = params.title;
  }
``` 
On line 101 `afterAll` and `afterEach` are not passed in when constructing the scenario. How do these tasks get passed into the scenario?

Reply:
True. Thanks!